src/glossary.adoc

```diff
@@ -77,7 +77,7 @@ See also _grouping mode_.
 grouping mode::
 One of three ways that a macro parameter is given arguments. A _simple parameter_ accepts one
 argument, a _grouped parameter_ accepts a list of arguments, and a _rest parameter_ accepts
-"all the rest" of the trailing arguments without a grouping list.
+“all the rest” of the trailing arguments without a grouping list.
```

Comment:
RE: dumb quotes, I believe my text editor has been using them by default throughout. Is there a reason you prefer smart quotes? Does your editor have a setting to enable them? I'm ok with either so long as we're consistent, though I'm not looking forward to the find-and-replace I'll need to do to update things.

This is not a blocking concern.

Reply:
> We have several typographic inconsistencies to remedy, but I've been inclined to deal with it later. 

Yeah, I think that's wise. I'd rather do it in as few passes as possible once we have content we're happy with. (O, to have an autoformatter!)

> I'm opinionated, so perhaps I should propose a basic style guide?

Sure, that'd be good.